ArduSub/ArduSub.cpp

```diff
@@ -341,13 +341,15 @@ void Sub::update_altitude()
 
 bool Sub::control_check_barometer()
 {
+    #if CONFIG_HAL_BOARD != HAL_BOARD_SITL
```

Comment:
Nitpicking: should be zero spaces for this macro, it's not inside any other pre-processor instruction.

Reply:
Sorry, but I did not get the issue here.
This line appears to be [the same one in others part of code](https://github.com/ArduPilot/ardupilot/search?utf8=%E2%9C%93&q=CONFIG_HAL_BOARD+%21%3D+HAL_BOARD_SITL&type=).
Can you give an example ? Or, if there is any, an astyle script or pattern to follow in ardupilot ?

Reply:
Sorry, I should have been more explicit. Issue is the spaces at beginning of the line, it shouldn't have any spaces.

Reply:
Oh, I see. Ty for points this, it's a pretty common style but I did not know that ArduPilot use it.
Done.